### DISCUSSION TITLE
Load multiple images and annotations in kornia

### DISCUSSION THREAD

**Discussion by UsmanAliKhan60** [kornia/kornia#2674] [Potential Issue]
Hi, I am working on kornia augmentation. I want to load all the images along with their bounding boxes collectively and augment them. Is their any function in kornia for this purpose. I have written a code which load all the images in one tensor and bounding boxes also in one tensor. Due to the fact that some images have more than one bounding boxes the size of images and bounding box tensor changes and their is no information in the tensor abount the number of bounding boxes in an image. How can I proceed with this?. Thanks

**Comment by edgarriba** [comment]
@UsmanAliKhan60 have you checked this ?https://kornia.readthedocs.io/en/latest/augmentation.container.html#kornia.augmentation.container.AugmentationSequential

not hundred percent sure we cover your case but for irregular samples you can always do iteratively sample per sample. Or find a strategy to put all your data in continuous memory to take advantage of vectorisation. Let us know, and any contribution in that direction is welcomed.
